src/pb_stub.cc

```diff
@@ -315,6 +382,68 @@ Stub::Initialize(bi::managed_external_buffer::handle_t map_handle)
   sys.attr("path").attr("append")(model_path_parent);
   sys.attr("path").attr("append")(model_path_parent_parent);
   sys.attr("path").attr("append")(python_backend_folder);
+  sys = py::module_::import(
+      (std::string(model_version_) + "." + model_name_trimmed).c_str());
+
+  py::module python_backend_utils =
+      py::module_::import("triton_python_backend_utils");
+  py::module c_python_backend_utils =
+      py::module_::import("c_python_backend_utils");
+  py::setattr(
+      python_backend_utils, "TritonError",
+      c_python_backend_utils.attr("TritonError"));
+  py::setattr(
+      python_backend_utils, "TritonModelException",
+      c_python_backend_utils.attr("TritonModelException"));
+  py::setattr(
+      python_backend_utils, "Tensor", c_python_backend_utils.attr("Tensor"));
+  py::setattr(
+      python_backend_utils, "InferenceRequest",
+      c_python_backend_utils.attr("InferenceRequest"));
+  py::setattr(
+      python_backend_utils, "InferenceResponse",
+      c_python_backend_utils.attr("InferenceResponse"));
+  c_python_backend_utils.attr("shared_memory") = py::cast(shm_pool_.get());
+
+  deserialize_bytes_ = python_backend_utils.attr("deserialize_bytes_tensor");
+  serialize_bytes_ = python_backend_utils.attr("serialize_byte_tensor");
+
+  return sys;
+}
+
+void
+Stub::AutoCompleteModelConfig(
+    bi::managed_external_buffer::handle_t string_handle,
+    std::string* auto_complete_config)
+{
+  py::module sys = StubSetup();
+
+  std::unique_ptr<PbString> pb_string_shm =
+      PbString::LoadFromSharedMemory(shm_pool_, string_handle);
+
+  py::module python_backend_utils =
+      py::module_::import("triton_python_backend_utils");
+  py::object model_config =
+      python_backend_utils.attr("ModelConfig")(pb_string_shm->String());
+
+  if (py::hasattr(sys.attr("TritonPythonModel"), "auto_complete_config")) {
+    model_config = sys.attr("TritonPythonModel")
+                       .attr("auto_complete_config")(model_config);
+  }
+
+  if (std::string(py::str(model_config.get_type())) !=
```

Comment:
I think it would be better to use `py::isinstance` instead of checking the string of the type.